### ISSUE TITLE
Turn exercise 008 into practice

### ISSUE THREAD

**Issue by cuihtlauac** [ocaml/ocaml.org#2176]
Extract the OCaml code from the markdown file corresponding to this exercise. The file is: `data/exercise/008_NAME.md` (where `008` is the number from the title of this issue and `NAME` is the rest of the file name, for instance `data/exercise/001_tail.md` is the file which corresponds to exercise `001`). Use the code in this file to create a folder `practice/008` (using this same number).

Populate the `008` folder with (almost) the same files as in folders `practice/001` and `practice/002` (they are examples).

Three files are different in each folder:
1. `practice/008/ex.ml`: it contains the function's signature, which needs to be implemented by the learner; 
1. `practice/008/work/impl.ml`: it includes the template meant to be filed by the learner;
1. `practice/008/answer/impl.ml`: consists of the solution from the original markdown file.

The examples included in the original markdown file must be turned into test cases in the `practice/008/ex.ml` file.

Folder structure to create:
```
practice/008/
├── answer
│   ├── impl.ml
│   └── test
│       ├── dune
│       └── run.ml
├── dune
├── dune-project
├── ex.ml
└── work
    ├── impl.ml
    └── test
        ├── dune

**Comment by komutumu** [comment]
Hello @cuihtlauac . 
Can I take this up please?

**Comment by cuihtlauac** [comment]
@jahielkomu: sure, please do

**Comment by komutumu** [comment]
> @jahielkomu: sure, please do

Thank you @cuihtlauac 
Please kindly expound on the statement; 
"Populate the 008 folder with (almost) the same files as in folders ```practice/001``` and ```practice/002```(they are examples)."

I'm failing to find these folders, that is,```practise/001``` and ```practise/002``` . Am I missing something?
Thanks.

**Comment by cuihtlauac** [comment]
Once you've cloned ocaml.org, the folders `001` and `002` are in the [`practice`](../tree/main/practice) folder. You must create your own (with the name `008`) aside from the existing ones.

**Comment by komutumu** [comment]
> Once you've cloned ocaml.org, the folders `001` and `002` are in the [`practice`](../tree/main/practice) folder. You must create your own (with the name `008`) aside from the existing ones.

I have to get the new changes locally, I cloned a forked repository way earlier let me get the updated version. 
Thank you

**Comment by komutumu** [comment]
Hello @cuihtlauac , I hope you are well.
Can I ask what the error ```No config found for file practice/008/answer/impl.ml. Try calling 'dune build'.ocamllsp``` means?
I have tried the suggestion therein but it hasn't helped.
If it is okay with you please advise.
Thank you.

**Comment by cuihtlauac** [comment]
Closed by #2242
